app/controllers/admin/users_controller.rb

```diff
@@ -1,25 +1,27 @@
-class Admin::UsersController < Admin::BaseController
-  before_action :set_user, only: [:show, :edit, :destroy]
-  def index
-    @users = User.client
-  end
+module Admin
+  class UsersController < Admin::BaseController
+    before_action :set_user, only: [:show, :edit, :destroy]
+    def index
+      @users = User.client
+    end
 
-  def show; end
+    def show; end
 
-  def edit; end
+    def edit; end
 
-  def update; end
+    def update; end
 
-  def destroy
-    if @user.destroy
-      redirect_to admin_users_path, notice: "User has been deleted"
-    else
-      flash.now[:alert] = 'Unable to delete user'
+    def destroy
+      if @user.destroy
+        redirect_to admin_users_path, notice: "User has been deleted"
+      else
+        flash.now[:alert] = 'Unable to delete user'
```

Comment:
use `redirect_to admin_users_path, alert: "Unable to delete user"`
Moreover, the path is the same. So set flash in if / else and redirect outside conditional block to refactor more